packages/mdc-tabs/_mixins.scss

```diff
@@ -0,0 +1,88 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import "@material/theme/mixins";
+@import "../variables";
+
+@mixin mdc-tab-label-ink-color($color) {
+  .mdc-tab:not(.mdc-tab--active) {
+    @include mdc-theme-prop(color, $color);
+  }
+
+  @include mdc-tab-icon-ink-color($color);
+}
+
+@mixin mdc-tab-icon-ink-color($color) {
+  .mdc-tab:not(.mdc-tab--active) .mdc-tab__icon {
+    @include mdc-theme-prop(color, $color);
+  }
+}
+
+@mixin mdc-tab-indicator-ink-color($color, $opacity: .87) {
+  .mdc-tab-bar__indicator {
+    @include mdc-theme-prop(background-color, $color);
+  }
+
+  &:not(.mdc-tab-bar-upgraded) .mdc-tab {
+    &:not(.mdc-tab--active):active::after {
+      opacity: $opacity;
+    }
+
+    &::after {
+      @include mdc-theme-prop(background-color, $color);
+    }
+  }
+}
+
+@mixin mdc-tab-label-activated-ink-color($color, $opacity: .87) {
+  @include mdc-tab-icon-activated-ink-color($color);
+
+  .mdc-tab.mdc-tab--active {
+    @include mdc-theme-prop(color, $color);
+  }
+
+  .mdc-tab:hover {
+    &,
+    .mdc-tab__icon {
+      @include mdc-theme-prop(color, $color);
+    }
+  }
+
+  &:not(.mdc-tab-bar-upgraded) .mdc-tab--active {
+    opacity: $opacity;
+  }
+}
+
+@mixin mdc-tab-icon-activated-ink-color($color) {
+  .mdc-tab.mdc-tab--active .mdc-tab__icon {
+    @include mdc-theme-prop(color, $color);
+  }
+}
+
+@mixin mdc-tab-indicator-height($height) {
```

Comment:
actually, don't bother with indicator height and width right now. just focus on the color mixins for now

Reply:
Ok will remove from PR